scripts/v2/aso-acr-protect.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```

Comment:
Is this script suppose to be used in task target `controller:acr-protect-image` ?

Reply:
No, it's a standalone script to protect sections of the acr. I used it to backfill protection